src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/ShowColumns.cs

```diff
@@ -118,7 +118,9 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
                     continue;
                 }
 
-                var columnName = new DName(expectedColumnName);
```

Comment:
This check should only be done if the support column names as identifiers is true

Reply:
Should we also change this in `DropColumn` function?

Reply:
Yep. I actually have a PR out that will merge the logic for both functions (#1959 ), we should wait until that is in before making this change.